sklearn/utils/sparsetools/setup.py

```diff
@@ -1,22 +1,15 @@
-import numpy
+# Remove in version 0.21
```

Comment:
Given that there is no tests anymore, I think you can remove the `tests` folder and `setup.py` as well. You only need to keep `__init__.py` for the deprecated `connected_components` function.